### PR TITLE
Clarify instructions re: frontend server config.

### LINKED ISSUE #164: What requests need to be forwarded to amppkg?
I'm trying to figure out which requests need to be forwarded from a front-end web server to amppkg.

Request for AMP resources (with the appropriate Accept header) should be forwarded to amppkg (amppkg will return application/signed-exchange), but it appears a few other endpoints also need to be forwarded too, such as `/priv/doc` and `/amppkg/validity` ([source](https://github.com/ampproject/amppackager/blob/1bea9db07a997ccc0a9ef451726e437aced37288/cmd/amppkg/main.go#L141-L144)).

Is this correct? Is there a definitive list of these endpoints somewhere?

### PR DESCRIPTION
**Pull request by twifkak**
Fixes #164.

### CODE REVIEW

**Comment by twifkak** [comment]
Please let me know if this needs something else to answer your question, @ithinkihaveacat.

File: README.md [hunk 1]
```diff
@@ -99,11 +99,15 @@ For now, productionizing is a bit manual. The minimum steps are:
...
   3. Configure your TLS-serving frontend server to conditionally proxy to
-     `amppkg`, if any of:
-     1. The URL starts with `/amppkg/`.
-     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
-        (Exercise caution not to send non-AMP URLs to `amppkg`; its
-        [transforms](transformer/) may break non-AMP pages.)
+     `amppkg`:
+     1. If the URL starts with `/amppkg/`, forward the request unmodified.
+     2. If the URL is AMP and the `AMP-Cache-Transform` request header is
+        present, rewrite the URL by prepending `/priv/doc` and foward the
```
Comment (ithinkihaveacat) [inline README.md]:
```suggestion
        present, rewrite the URL by prepending `/priv/doc` and forward the
```

**Comment by twifkak** [comment]
cla: @ithinkihaveacat is a Google employee.

File: README.md [hunk 2]
```diff
@@ -99,11 +99,15 @@ For now, productionizing is a bit manual. The minimum steps are:
...
      network.
   3. Configure your TLS-serving frontend server to conditionally proxy to
-     `amppkg`, if any of:
-     1. The URL starts with `/amppkg/`.
-     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
-        (Exercise caution not to send non-AMP URLs to `amppkg`; its
-        [transforms](transformer/) may break non-AMP pages.)
+     `amppkg`:
+     1. If the URL starts with `/amppkg/`, forward the request unmodified.
+     2. If the URL is AMP and the `AMP-Cache-Transform` request header is
```
Comment (alin04) [inline README.md]: Is saying a "URL is AMP" common terminology?

File: README.md [hunk 3]
```diff
@@ -99,11 +99,15 @@ For now, productionizing is a bit manual. The minimum steps are:
...
+        present, rewrite the URL by prepending `/priv/doc` and forward the
+        request.
+     3. Try not to send non-AMP URLs to `amppkg`; its [transforms](transformer/)
```
Comment (alin04) [inline README.md]: Instead of "try", maybe be more assertive and say "Don't" ?

File: README.md (hunk 2 above)
Comment (twifkak) [reply README.md]: Updated. It seems "AMP page" is accepted terminology [1], much like "ATM machine" or "PIN number".

[1] https://www.ampproject.org/docs/fundamentals/discovery

File: README.md (hunk 3 above)
Comment (twifkak) [reply README.md]: Hardened the language a bit. I still want to differentiate this from the `/priv/doc` thing, as the security risk of that is much greater.

File: README.md [hunk 4]
```diff
@@ -99,13 +99,18 @@ For now, productionizing is a bit manual. The minimum steps are:
...
-     1. The URL starts with `/amppkg/`.
-     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
-        (Exercise caution not to send non-AMP URLs to `amppkg`; its
-        [transforms](transformer/) may break non-AMP pages.)
+     `amppkg`:
+     1. If the URL starts with `/amppkg/`, forward the request unmodified.
+     2. If the URL points to an AMP page and the `AMP-Cache-Transform` request
+        header is present, rewrite the URL by prepending `/priv/doc` and forward
+        the request.
+     3. If at all possible, don't send non-AMP URLs to `amppkg`; its
```
[the remark this answers is not in the document]
Comment (twifkak) [reply README.md]: Done.

**Pull request merged**